common/src/test/java/org/apache/falcon/entity/parser/ProcessEntityParserTest.java

```diff
@@ -103,7 +103,7 @@ public void testParse() throws FalconException, JAXBException {
 
         Assert.assertEquals(process.getTags(),
                 "consumer=consumer@xyz.com, owner=producer@xyz.com, _department_type=forecasting");
-        Assert.assertEquals(process.getPipelines(), "testPipeline,dataReplication_Pipeline");
+        Assert.assertEquals(process.getPipelines(), "testPipeline");
```

Comment:
I have an assumption that one process can belong to only one pipeline but feed can belong to multiple pipelines.